src/mobile/mobile.go

```diff
@@ -110,12 +110,14 @@ func (m *Yggdrasil) StartJSON(configjson []byte) (conduit *dummy.ConduitEndpoint
 }
 
 // Stop the mobile Yggdrasil instance
-func (m *Yggdrasil) Stop() error {
+func (m *Yggdrasil) Stop() {
+	logger := log.New(m.log, "", 0)
+	logger.EnableLevel("info")
+	logger.Infof("Stop the mobile Yggdrasil instance %s", "")
+	//m.admin.Stop()
+	m.multicast.Stop()
+	//m.dummy.Stop()
```

Comment:
Why wouldn't we want the dummy adapter to be stopped?

Reply:
Could you please continue the optimization in other commit? I would like to focus on Android and Kotlin development.